project_0a/project_0a_6.c

```diff
@@ -0,0 +1,40 @@
+/*
+Project 0a
+*/
+
+/*
+Exercise 5 - build a multiplication table for a given number. Use scanf() to
+read input from keyboard. Limit previous implementation (exercise 5) to not
+allow negative values nor positive higher than 50
+
+Note: if statement is represented as:
+if (confition) {code for true condition} else {code for false condition}
+The else bit is optional
+*/
+
+#include <stdio.h>
+
+int main() {
+  int number, factor, calculation;
```

Comment:
Local variables should be declared as close to first use as possible. Initilization is prefered over declaration and assignment. They should be declared in the narrowest scope possible. https://google.github.io/styleguide/cppguide.html#Local_Variables